src/Features/Core/Portable/CodeFixes/Configuration/ConfigurationUpdater.cs

```diff
@@ -69,7 +69,6 @@ private enum ConfigurationKind
         private readonly string? _categoryToBulkConfigure;
         private readonly bool _isPerLanguage;
         private readonly Project _project;
-        private readonly CancellationToken _cancellationToken;
```

Comment:
Changes to this file seem unrelated to the core change, but I am not opposed to them going in, seems like good cleanup